Platform.PCL/Realm.PCL/RealmObjectPCL.cs

```diff
@@ -503,11 +503,5 @@ protected void RaisePropertyChanged([CallerMemberName] string propertyName = nul
         protected virtual void OnPropertyChanged(string propertyName)
         {
         }
-
-        TypeInfo IReflectableType.GetTypeInfo()
```

Comment:
They are - the new package Realm.Databinding contains the logic for automatic transactions and the `IReflectableType` implementation is woven to all user types.